interceptor/src/twcc/mod.rs

```diff
@@ -57,11 +57,10 @@ impl Recorder {
     /// build_feedback_packet creates a new RTCP packet containing a TWCC feedback report.
     pub fn build_feedback_packet(&mut self) -> Vec<Box<dyn rtcp::packet::Packet + Send + Sync>> {
         let mut feedback = Feedback::new(self.sender_ssrc, self.media_ssrc, self.fb_pkt_cnt);
```

Comment:
should we move this down (after if condition)?

Reply:
Indeed we should